### PR TITLE
Test on newly installed u2004VM

### PR DESCRIPTION
**Pull request by Hustwireless**
1. fix catkin: command not found issue
2. fix a minor typo
3. modified installation step of spinnaker driver (based on former installation on my machine)

### CODE REVIEW

File: docs/robot-operation/installation_drivers.md [hunk 1]
```diff
@@ -65,12 +65,8 @@ cd spinnaker-2.4.0.143-amd64/
 sudo dpkg -i libgentl_2.4.0.143_amd64.deb libspinnaker_2.4.0.143_amd64.deb libspinnaker-dev_2.4.0.143_amd64.deb libspinnaker-c_2.4.0.143_amd64.deb libspinnaker-c-dev_2.4.0.143_amd64.deb
 
 # Setting up FLIR driver (libspinnaker). Enter username when asked!
-sudo ./configure_gentl_paths.sh 64
-sudo ./configure_usbfs.sh
-sudo ./configure_spinnaker.sh
-sudo ./configure_spinnaker_paths.sh
+sudo ./install_spinnaker.sh
```
Comment (mantelt) [inline docs/robot-operation/installation_drivers.md:68]: The point of not using the install_spinnaker.sh is that this script installs more than actually needed (e.g. libspinvideo*)

File: docs/core-software/installation_core.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ catkin config -DCMAKE_BUILD_TYPE=Release
 
 If there is an error of "*catkin: command not found*" 
 ```bash
-sudo apt install python3-catkin-lint python3-pip
+sudo apt install python3-catkin-tools python3-catkin-lint python3-pip
```
Comment (mantelt) [inline docs/core-software/installation_core.md:68]: good catch, thanks!

**Review by mantelt — CHANGES_REQUESTED** [review]
Thanks a lot for going though all of this!
As pointed out, I would still use the individual configure scripts instead of the `install_spinnaker.sh`. I think that should work as well, doesn't it?

File: docs/robot-operation/installation_drivers.md (hunk 1 above)
Comment (Hustwireless) [reply docs/robot-operation/installation_drivers.md:68]: Yes, I see the point here and agree with you.

**Pull request merged**